src/js/types/account.js

```diff
@@ -173,12 +173,14 @@ export type AccountInfo = {
 export type RegularOutput = {
     type?: 'external',
     address: string,
+    address_n?: typeof undefined,
```

Comment:
`address_n` account.d.ts? or maybe this is just some flow union magic right?

Reply:
all `typeof undefined` in type unions are required only by flow